### PR TITLE
Add Comment for Describe

### PR DESCRIPTION
**Pull request by masitings**

### CODE REVIEW

File: config/browserlists.js [hunk 1]
```diff
@@ -6,6 +7,7 @@ const allSupportedBrowsers = [
   'ie 11',
 ];
 
+// Browser that can adapt and running
```
Comment (rwjblue) [inline config/browserlists.js]: This isn't quite right, this is the set of browsers we use for the prebuilt assets (though the application can configure whichever ones it wants).

File: config/browserlists.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// The list of supported browsers
```
Comment (rwjblue) [inline config/browserlists.js]: I'd tweak this to say `The list of all supported browsers`

**Review by rwjblue — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left a few inline comments for tweaks.

Also wanted to mention, that its _generally_ not a good idea to send PR's from your forks `master` branch directly. Usually you'd want to make a branch in your fork and send the PR from there. No worries in this case, but figured I'd mention it in case you weren't familiar with the workflow. 😸

File: config/browserlists.js (hunk 2 above)
Comment (Alonski) [reply config/browserlists.js]: Do we really need this though?
A variable named `fullName` does not need a comment above it that says `The full name` right?

**Comment by MelSumner** [comment]
This is not a valid PR. Closing issue.

**Pull request closed without merging**